mods/optifine.json

```diff
@@ -0,0 +1,4775 @@
+{
+    "format": 1,
+    "name": "OptiFine",
+    "authors": ["sp614x"],
+    "disc": "Minecraft optimization mod",
+    "discLong": "This mod adds support for HD textures and a lot of options for better looks and performance. Doubling the FPS is common.",
```

Comment:
There is no `descLong`. Long descriptions go in `desc`. There may be a `shortDesc` in the future, but not yet.